pkgs/by-name/gi/git-gamble/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitLab,
+  git,
+  installShellFiles,
+  makeWrapper,
+  nix-update-script,
+}:
+
+let
+  version = "2.9.0";
+
+  src = fetchFromGitLab {
+    owner = "pinage404";
+    repo = "git-gamble";
+    rev = "version/${version}";
+    hash = "sha256-hMP5mBKXcO+Ws04G3OxdYuB5JoaSjlYtlkerRQ6+bXw=";
+  };
+in
+rustPlatform.buildRustPackage {
+  pname = "git-gamble";
+  inherit version src;
+
+  cargoHash = "sha256-vrzcNdLY2PkyZ1eLwOiONRHVAolbTDxytEgi09WkDZQ=";
+
+  nativeCheckInputs = [ git ];
+  preCheck = ''
+    patchShebangs tests/editor/fake_editor.sh
+  '';
+
+  nativeBuildInputs = [
+    installShellFiles
+    makeWrapper
+  ];
+  postInstall = ''
+    wrapProgram $out/bin/git-gamble \
+      --prefix PATH : "${lib.makeBinPath [ git ]}"
```

Comment:
i wonder which is better between `git` and `gitMinimal` ?

Reply:
Looking at the [package definition of `gitMinimal`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix#L2145-L2150) I'd say it's a good choice to use here, unless git-gamble requires support for python, perl or pcre2 in git…

Reply:
Done !

Thanks for the quick response !